deploy/deploy.js

```diff
@@ -11,7 +14,8 @@ async function deployNativeToErc(){
   console.log(`[   Home  ] HomeBridge: ${homeBridge.address} at block ${homeBridge.deployedBlockNumber}`)
   console.log(`[ Foreign ] ForeignBridge: ${foreignBridge.address} at block ${foreignBridge.deployedBlockNumber}`)
   console.log(`[ Foreign ] POA20: ${erc677.address}`)
-  fs.writeFileSync('./bridgeDeploymentResults.json', JSON.stringify({
+  const deployResultsPath = path.join(__dirname, './bridgeDeploymentResults.json')
```

Comment:
Should you do the same for `deployErcToErc()`?

Reply:
Oh, I missed that, thanks!